src/CodingMilitia.PlayBall.Shared.StartupTasks/DbInitializer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AspNetCore.AsyncInitialization;
+using CodingMilitia.PlayBall.GroupManagement.Data;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+
+namespace CodingMilitia.PlayBall.Shared.StartupTasks
+{
+    public class DbInitializer : IAsyncInitializer
+    {
+        private IWebHost _host;
+        public DbInitializer(IWebHost host)
```

Comment:
As we register the initializer in the dependency injection container, we can receive our required dependencies directly in the constructor instead of `IWebHost`, which I was using to have access to the `IServiceProvider`.

So I would say the constructor can change to something like:
```csharp
public DbInitializer(T dbContext, IHostingEnvironment hostingEnvironment)
//...
```